go/api/channel.go

```diff
@@ -1,4 +1,4 @@
-package resource
+package api
```

Comment:
I wonder if this file would be better placed in `pkg/channel` since that is the only placed it is used?